src/Microsoft.Security.Utilities.Core/PreciselyClassifiedSecurityKeys/IdentifiableKey.cs

```diff
@@ -53,14 +53,36 @@ public override IEnumerable<string> GenerateTruePositiveExamples()
                     string encoded = new string(alphabet[i], encodedLength);
                     Array.Copy(Convert.FromBase64String(encoded), bytes, KeyLength);
 
-                    yield return
+                    string key =
                         IdentifiableSecrets.GenerateBase64KeyHelper(checksumSeed,
                                                                     keyLengthInBytes: KeyLength,
                                                                     Signatures!.First(),
                                                                     EncodeForUrl,
                                                                     bytes);
+
+                    yield return key;
+
+                    foreach (string prefix in s_nonInvalidatingPrefixes)
+                    {
+                        yield return $"{prefix}{key}";
+                    }
+
+                    foreach (string suffix in s_nonInvalidatingSuffixes)
+                    {
+                        yield return $"{key}{suffix}";
+                    }
                 }
             }
         }
+
+        private static readonly string[] s_nonInvalidatingPrefixes = new[]
```

Comment:
>s_nonInvalidatingPrefixes [](http://example.com/codeflow?start=41&length=25)

i've included the characters that were observed in the wild to provoke an issue. presumably, fixing these fixes the full range of improperly performing chars in play, since we no longer have hyphen literal vs. range operator confusion.

and so, I didn't bother to invest anything in trying to produce a comprehensive set of things to test here.